studio/components/interfaces/BranchManagement/Overview.tsx

```diff
@@ -1,4 +1,4 @@
-import { useParams } from 'common'
+import { noop } from 'lodash'
```

Comment:
```suggestion
import { noop } from 'lib/void'
```
reduce deps = make Kevin happy? 🤷